plugins/patterns/patterns_inspector.rb

```diff
@@ -60,11 +60,16 @@ def inspect_with_zypper
         xml = @system.run_command("zypper", "--non-interactive", "-xq", "--no-refresh", "patterns",
           "-i", stdout: :capture)
       rescue Cheetah::ExecutionFailed => e
-        if e.status == 7  # ZYPPER_EXIT_ZYPP_LOCKED
+        # Zypper is locked
+        if e.status.exitstatus == 7 # ZYPPER_EXIT_ZYPP_LOCKED
           Machinery.logger.error(e.stdout)
           raise Machinery::Errors::ZypperFailed.new(
             "Zypper is locked."
           )
+        # Repositories are updated or gpg keys are expiring
+        elsif e.status.exitstatus == 106 # ZYPPER_EXIT_INF_REPOS_SKIPPED
+          Machinery.logger.error(e.stdout)
+          xml = e.stdout
```

Comment:
Those two lines sound a bit contradicting. Either the command succeeded and the result can be used as the `xml` variable or it failed and should be logged and treated as an error, no?